### ISSUE TITLE
Application is stopped with Error/Exception when Loki is not avaiable

### ISSUE THREAD

**Issue by kambei** [loki4j/loki-logback-appender#74]
When the appender fails to send logs to loki, either because loki is not available or because the host is wrong, it fails and stops the application!

Instead it might just to "inform", by making the application continue to run.

**Comment by nehaev** [comment]
Hi and thanks for reporting!

If Loki is unavailable or an exception happened while processing response from Loki, appender reports an error to stderr. It does not stop the application and even does not stop itself. If the behavior you observe is different from this, please provide the details that will help to reproduce the issue:

- Version of Loki4j, version of JDK and OS you are running on
- Exact steps which reproduce the problem in as many details as possible
- Code snippets that help to reproduce a problem and don't contain any irrelevant parts
- Uncut stacktraces copied with all the nested exceptions
- Your Loki4j configuration and logs (that are written to `stderr`)
- Specify if you use frameworks or application servers that could affect logging lifecycle (e.g. Tomcat, Spring Boot, etc.)

**Comment by kambei** [comment]
Hi,
I encountered the error developing an application for which I cannot provide the code and logs.

Anyway,
I am using Spring Boot, openjdk-11 and the version of loki-logback-appender is the latest (v1.0.0).

I tried to replicate the error in [loki-logback-appender-test](https://github.com/kambei/loki-logback-appender-test):
if the application is launched (without having Loki available), maybe you make some calls on the endpoint `/test`, after a while some exceptions appears, but the application does not stop. As you said.

The error/exception is the same one that in my enterprise project caused the application not to start.
So, at this point, returning to my enterprise project, after a `gradle clean`, restarting everything from a "clean status" was enough to no longer have the error and have the application up. Despite Loki's unavailability.

So I believe I had dirty and invalidated gradle cache for some reason.

Sorry for the waste of time and this useless issue.
